### DISCUSSION TITLE
feasibility of including the host in the community model

### DISCUSSION THREAD

**Discussion by mmp3** [micom-dev/micom#42] [Ideas]
Hi!

Thank you for this wonderful tool!

For microbial communities that exist within a host, such as the human gut microbiome, **would it make sense to include the host (e.g. human) as a community member in the `MiCOM` model?**

A lot of effort has gone into reconstructing metabolic models for human cells, even with tissue-specificity, so it would seem beneficial to take incorporate metabolic models for human cells, when appropriate. 

With respect to the `MiCOM` framework, it is not immediately obvious how to set the abundance of a "host" community member. For example, metagenomics samples from the human gut microbiome may consist of ~30% human DNA, whereas metagenomics samples from the human vaginal microbiome are often >99% human DNA. Would it make sense to use these fractions as the "abundance" of the host community member in a hypothetical `MiCOM` model? **Otherwise, how should host abundance be chosen?**

Additionally, my understanding is that growth rates are at the core of `MiCOM` and FBA more generally. **But are host (e.g. human) cells considered to be growing in the same sense as the growth of the various microbial populations?** For instance, does human gut epithelial cell proliferation represent the same kind of growth as that of microbes modeled by `MiCOM` and FBA?

**Comment by cdiener** [comment]
You are very welcome :smile:

Would it make sense? Absolutely! I think you did a great job of identifying the challenges though. 

Abundances are the first one. Host DNA in metagenomics probably comes from dead or lysed cells in the gut and not really from the intestinal cells which are intact and likely don't release a lot of DNA. I don't think we have a good estimate of the ratio of bacterial to intestinal cells' biomass in the distal gut. It's probably biased towards intestinal cells though. 

The objective is the second challenge. If intestinal cells would grow like bacteria they would be cancerous. What is usually done in the field is to define a maintenance function (required metabolic conversions that need to be fulfilled by the tissue) and use that instead of growth. You could try to include this into the MICOM objective. 

The third challenge is to actually have good models of the various intestinal cell types in the human gut and their maintenance objectives. There are some approaches like the work from Ines Thiele's lab on the [Harvetta and Harvey models](https://www.embopress.org/doi/full/10.15252/msb.20198982).

**Reply by mmp3** [reply]
I see. That is what I was afraid of :)
Thank you!

**Reply by cdiener** [reply]
Yep, that does not mean that it is impossible though. One only needs to find a strategy to deal with those points. We have a PhD student in the Gibbons Lab that is working on that and there are other groups working on that as well (maybe not with MICOM specifically though).
